oak/server/rust/oak_abi/src/lib.rs

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright 2019 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+//! Type and constant definitions for the Oak application binary interface.
+
+pub use proto::oak_api::ChannelReadStatus;
+pub use proto::oak_api::OakStatus;
+
+pub mod proto;
+
+/// Handle used to identify read or write channel halves.
+///
+/// These handles are used for all host function calls.
+pub type Handle = u64;
```

Comment:
Tracking comment to make sure we don't end up with 3 handle types. I think this will be solved during rebasing

Reply:
Yep, done in the rebase.